src/clients/Deluge.ts

```diff
@@ -140,34 +168,38 @@ export default class Deluge implements TorrentClient {
 	 * parses the set-cookie header and updates stored value
 	 */
 	private handleResponseHeaders(headers: Headers) {
-		if (headers.has("Set-Cookie")) {
-			this.delugeCookie = headers.get("Set-Cookie").split(";")[0];
-		}
+		this.delugeCookie = headers.has("Set-Cookie")
+			? headers.get("Set-Cookie").split(";")[0]
+			: null;
```

Comment:
I don't think this change is beneficial

Reply:
My thought was if the header doesn't container the cookie, which it will every response as long as the cookie is valid, we should invalidate it.

I guess the one aspect that I didn't consider is transient errors mid-stream?

Either way, if we nullify the cookie header `call` will automatically reauthenticate and cookie should thus be reset and things should proceed normally.

Reply:
yeah, but the way that cookies work more generally is that they are expected to persist until they expire or the server sends another Set-Cookie header. 

If deluge changed their implementation on the server to send the Set-Cookie header for only the login endpoint, the UI would still work just fine because browsers cache cookies. But with this change, cross-seed would require a reauth for every call

Reply:
Gotcha. Futureproofing.